src/seedu/addressbook/commands/HelpCommand.java

```diff
@@ -25,6 +25,7 @@ public CommandResult execute() {
                 + "\n" + ViewAllCommand.MESSAGE_USAGE
                 + "\n" + HelpCommand.MESSAGE_USAGE
                 + "\n" + ExitCommand.MESSAGE_USAGE
+                        + "\n" + InfoCommand.MESSAGE_USAGE
         );
```

Comment:
It is better to be consistent with the alignments. Though it is a minor change it is not a good practice to avoid it.